modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java

```diff
@@ -2681,37 +1522,28 @@ private MockLiferayPortletRenderRequest _getMockLiferayPortletRenderRequest(
 	}
 
 	private ServiceContext _getServiceContext(
-			long userId, long groupId, long[] assetCategoryIds)
+			long userId, long groupId, long[] assetCategoryIds,
+			String[] assetTagNames)
 		throws PortalException {
 
 		ServiceContext serviceContext =
 			ServiceContextTestUtil.getServiceContext(groupId, userId);
 
 		serviceContext.setAssetCategoryIds(assetCategoryIds);
+		serviceContext.setAssetTagNames(assetTagNames);
```

Comment:
What do you think about create a new method in ServiceContextTestUtil, that create a ServiceContext with these parameters?

Reply:
I think It would be useful, at least for us ;)

Reply:
@cgoncas, after reviewing the `ServiceContextTestUtil` code, adding a new method with params for assets would be a bit inaccurate since IMHO the ServiceContextTestUtil should remain simple as it is, only with company, group and user. Other than that breaks the "conventions" followed in this class. But, here we are, ready to break the rules! Let me know what you think. Thanks!

Reply:
There is no any convention in ServiceContextTestUtil, Manu just create the methods that were used in the code at that point.

Reply:
ahh, perfect, so, I'm try to add it then!

Reply:
Added it, thanks!